lib/elixir/test/elixir/task_test.exs

```diff
@@ -495,6 +495,16 @@ defmodule TaskTest do
              {:timeout, {Task.Supervised, :stream, [0]}}
       refute_received _
     end
+
+    test "streams an enumerable with ordered: false" do
```

Comment:
On slack we said we should use the element value instead of the position. The element value is more versatile because you can always get the position with `Stream.with_index` and sometime the element value might be more useful. The downside of using the element is that you keep it in memory for longer. What do you think we should use?

I am using the element value for now. Let me know if I should change it.